crates/dap_adapters/src/go.rs

```diff
@@ -437,18 +439,44 @@ impl DebugAdapter for GoDebugAdapter {
             adapter_path.join("dlv").to_string_lossy().to_string()
         };
 
-        let cwd = task_definition
-            .config
-            .get("cwd")
-            .and_then(|s| s.as_str())
-            .map(PathBuf::from)
-            .unwrap_or_else(|| delegate.worktree_root_path().to_path_buf());
+        let cwd = Some(
+            task_definition
+                .config
+                .get("cwd")
+                .and_then(|s| s.as_str())
+                .map(PathBuf::from)
+                .unwrap_or_else(|| delegate.worktree_root_path().to_path_buf()),
+        );
 
         let arguments;
         let command;
         let connection;
 
         let mut configuration = task_definition.config.clone();
+
+        let mut envs = HashMap::default();
+        if let Some(env_file) = configuration.get("envFile").and_then(Value::as_str) {
```

Comment:
I believe `envFile` could also be an array of paths:
https://github.com/search?q=repo%3Agolang%2Fvscode-go%20envfiles&type=code